content/influxdb/cloud-dedicated/reference/client-libraries/v2/javascript/nodejs/write.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Write data with the InfluxDB JavaScript client library
+description: >
+  Use the JavaScript client library to write data with the InfluxDB API in Node.js.
+menu:
+  influxdb_cloud_dedicated:
+    name: Write
+    parent: Node.js
+influxdb/cloud-dedicated/tags: [client libraries, JavaScript]
+weight: 101
+aliases:
+  - /influxdb/cloud-dedicated/reference/api/client-libraries/nodejs/write
+related:
+  - /influxdb/cloud-dedicated/write-data/troubleshoot/
+---
+
+Use the [InfluxDB Javascript client library](https://github.com/influxdata/influxdb-client-js) to write data from a Node.js environment to InfluxDB.
+
+The Javascript client library includes the following convenient features for writing data to InfluxDB:
+- Apply default tags to data points.
+- Buffer points into batches to optimize data transfer.
+- Automatically retry requests on failure.
+- Set an optional HTTP proxy address for your network.
+
+### Before you begin
+
+- [Install the client library and other dependencies](/influxdb/cloud-dedicated/api-guide/client-libraries/nodejs/install/).
+### Write data with the client library
+
+1. Instantiate an `InfluxDB` client. Provide your InfluxDB URL and API token.
+
+   ```js
+   import {InfluxDB, Point} from '@influxdata/influxdb-client'
+
+   const influxDB = new InfluxDB({YOUR_URL, YOUR_API_TOKEN})
+   ```
+   Replace the following:
+   - *`YOUR_URL`*: InfluxDB URL
+   - *`YOUR_API_TOKEN`*: InfluxDB API token
+
+2. Use the `getWriteApi()` method of the client to create a **write client**.
+   Provide your InfluxDB organization ID and bucket name.
+
+   ```js
+   const writeApi = influxDB.getWriteApi(YOUR_ORG, YOUR_BUCKET)
+   ```
+   Replace the following:
+   - *`YOUR_ORG`*: InfluxDB organization ID
+   - *`YOUR_BUCKET`*: InfluxDB bucket name
```

Comment:
@sanderson Thanks for fixing these. You're right, of course. I was only thinking about it in 2.x context for the list. Otherwise, I would have hidden it for now.